src/components/pages/rules/flyouts/ruleEditor/ruleEditor.js

```diff
@@ -40,8 +46,7 @@ import {
   toDiagnosticsModel,
   toSinglePropertyDiagnosticsModel
 } from 'services/models';
-import Config from 'app.config';
-import update from 'immutability-helper';
+
```

Comment:
nit: extra blank line

Reply:
Actually, its a convention used throughout the application that we put a blank line between the 3rd party stuff and our stuff in the import lists.